docs/ops/generation/EyeLike_9.md

```diff
@@ -0,0 +1,142 @@
+## EyeLike <a name="EyeLike"></a> {#openvino_docs_ops_generation_EyeLike_9}
+
+**Versioned name**: *EyeLike-9*
+
+**Category**: *Generation*
+
+**Short description**: *EyeLike* operation generates shift matrix.
```

Comment:
```suggestion
**Short description**: The *EyeLike* operation will generate shift matrices.
```

Reply:
I see that almost all specifications have a template of a short description :
`*NAME* operation generates` without _Future Simple_ and article _the_.
Moreover one operation generates one matrix not several matrices. It's important
So I'd rather leave it as it is.
Thanks!